Mathlib/GroupTheory/HNNExtension.lean

```diff
@@ -175,11 +175,11 @@ variable (G A B)
 coset of both `A` and `B`, such that the chosen element of the subgroup itself is `1`. -/
 structure TransversalPair : Type _ :=
   /-- The transversal of each subgroup -/
-  ( set : ℤˣ → Set G )
```

Comment:
#7359 removes the parentheses entirely

Reply:
Good point. I'll merge master and fix accordingly.